ChatApp/ChatView/MessageCell.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  MessageCell.swift
+//  ChatApp
+//
+//  Created by Giorgi Makadze on 21.04.2023.
+//
+
+import Foundation
+import UIKit
+
+class MessageCell: UICollectionViewCell {
+    let label = UILabel()
+    let conteinerView = UIView()
+    //    let elipse = UIView()
+    let textDate = UILabel()
+    
```

Comment:
add "// MARK: Init". Such marks will help you to easily navigate throughout the file.